### DISCUSSION TITLE
Gledopto GL-FL-004TZ - not supported

### DISCUSSION THREAD

**Discussion by volker73** [Koenkk/zigbee2mqtt#13680] [Devices]
Am unfortunately a beginner in this area, although already have a few ZigBee devices in operation and all work with On / Off, color change etc..

I have now bought a LED floodlight "Gledopto GL-FL-004TZ", for which there is also a device sheet on your site.
There are apparently also several who have this spotlight successfully in operation.

The teach-in (connect) was no problem, it was also recognized.
I tried to switch the light on and off in different ways (JSON {"state: "on"}) but without success.

It appears constantly the message:
**"Device with modelID 'GL-FL-004P' is not supported."**

After the factory setting, there is a green light in the center of the emitter, but have not found any posts as to what this means.

I have also found the "Support new device" page. But this is beyond my capabilities!

I have a "RED MATIC 7.1.2" running on a RaspberryMatic.
I use "Z-Stack/ ZigBee Shepard"

What am I doing wrong?

kind regards
Volker

**Comment by Koenkk** [comment]
You are probably using an old z2m version, this device is supported in zigbee2mqtt 1.27.0

**Comment by volker73** [comment]
Hello,

thanks for the answer.

As mentioned, I am a beginner.

I use RED NODE
and the ZigBee Node
   - node-red-contrib-zigbee 0.20.0

Which Node should I use?
  - node-red-contrib-zigbee2mqtt 2.6.3 for example?
  There are are lot of dependencies which I have to install before
Or am I on the wrong track?


*Volker*


Am Do., 25. Aug. 2022 um 17:30 Uhr schrieb Koen Kanters <
***@***.***>:

**Reply by Koenkk** [reply]
`node-red-contrib-zigbee` is a different project, it does not use Zigbee2MQTT. I suggest to create an issue in this repo: https://github.com/hobbyquaker/node-red-contrib-zigbee and ask if he can update the zigbee-herdsman-converters dependency.

**Comment by volker73** [comment]
Thank you for your help,

sorry I was not at home during the weekend.
I received you mail and your solution.
I have opened a new issue one the page you told me.

Volker


Am Fr., 26. Aug. 2022 um 16:56 Uhr schrieb Koen Kanters <
***@***.***>:
